JavaScript/JavaScript.sublime-syntax

```diff
@@ -25,205 +41,197 @@ contexts:
   prototype:
     - include: comments
 
+  comments:
+    - match: /\*\*(?!/)
+      scope: punctuation.definition.comment.js
+      push:
+        - meta_include_prototype: false
+        - meta_scope: comment.block.documentation.js
+        - match: \*/
+          scope: punctuation.definition.comment.js
+          pop: true
+    - match: /\*
+      scope: punctuation.definition.comment.js
+      push:
+        - meta_include_prototype: false
+        - meta_scope: comment.block.js
+        - match: \*/
+          scope: punctuation.definition.comment.js
+          pop: true
+    - match: //
+      scope: punctuation.definition.comment.js
+      push:
+        - meta_include_prototype: false
+        - meta_scope: comment.line.double-slash.js
+        - match: \n
+          pop: true
+
+  comments-top-level:
+    - match: ^(#!).*$\n?
+      scope: comment.line.shebang.js
+      captures:
+        1: punctuation.definition.comment.js
+
+  else-pop:
+    - match: (?=\S)
+      pop: true
+
+  immediately-pop:
+    - match: ''
+      pop: true
+
+  comma-separator:
+    - match: ','
+      scope: punctuation.separator.comma.js
+
   keywords-top-level:
     - match: \bimport\b
-      scope: meta.import.js keyword.control.import-export.js
-      push: import-extended
+      scope: keyword.control.import-export.js
+      push:
+        - import-meta
+        - import-export-final
+        - import-extended
+
     - match: \bexport\b
-      scope: meta.export.js keyword.control.import-export.js
-      push: export-extended
+      scope: keyword.control.import-export.js
```

Comment:
I wouldn't like to say `import.export` because that to me implies that the keyword is of subtype `import` and sub-subtype `export`. The keyword may be `import` or `export`, but it is not both, and `export` is not truly subsidiary to `import`.